.github/workflows/automatic-updates.yml

```diff
@@ -0,0 +1,37 @@
+name: Automatically update Docker image versions
+
+on:
+  schedule:
+    - cron: "15 * * * *"
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    if: github.repository_owner == 'iasuhiu3hdiuwdhiuwhqfs'
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-node@v2
+        with:
+          node-version: 'lts/*'
+
+      - name: Run automation script
+        run: node ./build-automation.js
+
+      - name: Create update PR
+        uses: peter-evans/create-pull-request@v3
+        with:
+          token: ${{ secrets.GH_API_TOKEN }}
+          branch: update-branch
+          commit-message: "version update"
+          title: "Version update"
+          body: |
+            @LaurentGoderre @PeterDaveHello @SimenB @Starefossen @nschonni @ttshivers
```

Comment:
can we do `@nodejs/docker`? since it'll be opened in this repo I assume the team tag will work.

Or even better: use `team-reviewers`: https://github.com/peter-evans/create-pull-request#action-inputs